spec/index.bs

```diff
@@ -3878,6 +3853,56 @@ Issue: link to the semantics of SV_SampleIndex and SV_Coverage in WGSL spec.
         </div>
 </dl>
 
+<div algorithm>
+    <dfn abstract-op>validating GPURenderPipelineDescriptor</dfn>(descriptor, features)
+        **Arguments:**
+            - {{GPURenderPipelineDescriptor}} |descriptor|
+            - [=list=]&lt;{{GPUFeatureName}}&gt; |features|
```

Comment:
nit: what about limits?

Reply:
not needed currently for this algorithm, we'll add it when it's needed